src/uwtools/scheduler.py

```diff
@@ -144,5 +161,33 @@ class LSF(JobScheduler):
     }
 
 
+@shelve_it("cache.shelve")
+def submit_job_card(content) -> int:
+    """sends the job card to the scheduler for processing"""
+    p = subprocess.Popen(
+        ["sbatch", "--deadline=now+1"],
```

Comment:
This is a slurm-specific submission. We will likely need the submission string to be provided for us from the scheduler subclasses.